crates/proof-of-sql/src/proof_primitive/hyrax/mod.rs

```diff
@@ -0,0 +1,5 @@
+/// The hyrax scheme in generic form.
+pub(super) mod base;
+/// A valid scheme that can be used for testing. This could become its own scheme. It's a reasonable scheme.
```

Comment:
This comment doesnt make a lot of sense

Reply:
Made a change. Should be more clear now.